lib/engines/content_block_manager/app/forms/content_block_manager/content_block/edition_form.rb

```diff
@@ -15,8 +15,15 @@ def initialize(content_block_edition:, schema:)
     @schema = schema
   end
 
+  def remove_internal_metadata(schema_fields)
+    schema_fields.delete("instructions_to_publishers")
+    schema_fields
+  end
+
   def attributes
-    @schema.fields.each_with_object({}) do |field, hash|
+    puts "fields are in form #{@schema.fields}"
```

Comment:
Is this some left over debug code?

Reply:
oh yep this is when I was experimenting with putting the field in `details`

Reply:
removed 👍